src/GlobalX.ChatBots.WebexTeams/Services/HttpClientProxy.cs

```diff
@@ -70,7 +70,10 @@ public async Task DeleteAsync(string path)
             };
 
             var response = await _httpClient.SendAsync(request).ConfigureAwait(false);
-            response.EnsureSuccessStatusCode();
+            if (response.StatusCode != HttpStatusCode.NotFound)
+            {
+                response.EnsureSuccessStatusCode();
```

Comment:
so basically, when a DELETE request is made, we succeed if there is a 2XX or a 404 response, and otherwise we throw an exception?

Why only on DELETEs?

Reply:
This wouldn't make sense on a POST request I think. Nor really on a GET, as you should probably know that the thing exists before trying to GET it (and if the 404 wasn't a problem on a GET, the inability to deserialize any content would be).

This is really just to fix an issue where a webhook that shows up on a GET call can't be DELETEd